test/unit/helpers/recruitment_banner_helper_test.rb

```diff
@@ -0,0 +1,49 @@
+require "test_helper"
+
+class RecruitmentBannerHelperTest < ActionView::TestCase
+  include RecruitmentBannerHelper
+
+  def setup
+    @banner_data = {
```

Comment:
Could we use a yaml fixture here instead? I think it would be nice to test the parsing code, given the structure of the banners array is really important to this working.